.github/CONTRIBUTING.md

```diff
@@ -81,6 +99,20 @@ $ make html
 
 The built documentation can then be found in `docs/_build/html/`.
 
+To file a pull request, create a new branch on top of the upstream repository:
+
+```console
+$ git checkout -b my_topical_branch upstream/main
+```
+
+Make your changes, push them to your fork (the remote *origin*) and publish the PR!
```

Comment:
This could be accompanied with the following snippet:
```console
$ git push -u origin feature/my-topic-branch
```
`-u` marks the branch on the `origin` remote (aka fork) as corresponding to the local one so that `git push` w/o args would pick it up automatically.

Reply:
i removed the branch name because it feels tedious and it's unclear to me what problem it's supposed to solve. happy to be corrected, but I'll try to keep it as simple as possible.